### PR TITLE
Model SKU updates for model evaluation

### PR DESCRIPTION
**Pull request by sarthaks95**
Modifying recommended SKU list for models to be >= minimum SKUs for Model Evaluation

Changes made based on valdn wf: https://github.com/Azure/azureml-assets/actions/runs/7444059908/job/20249849771?pr=2079

### CODE REVIEW

File: assets/models/system/tiiuae-falcon-40b/spec.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ properties:
   SHA: 3d7c5902f1dc9da830979a826cd96114b3ba4ec1
   datasets: tiiuae/falcon-refinedweb
   languages: en, de, es, fr
```
Comment (BhavanaTumma) [inline assets/models/system/tiiuae-falcon-40b/spec.yaml:8]: is this value correct? 24 cores, 448GB RAM, 1344GB storage --> this is what I see

Comment (BhavanaTumma) [reply assets/models/system/tiiuae-falcon-40b/spec.yaml:8]: or do we mean we need 4 instances of this compute to run?

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 19308160448536e378e3db21a73a751579ee7fdd
-  evaluation-min-sku-spec: 24|4|448|1344
+  evaluation-min-sku-spec: 24|1|220|64
   evaluation-recommended-sku: Standard_NC24s_v3, Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_NC96ads_A100_v4
```
Comment (BhavanaTumma) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:8]: Can you check the value here? evaluation-min-sku-spec

**Comment by novaturient95** [comment]
Could you please add description and why this change is needed?

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 2 above)
Comment (BhavanaTumma) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:8]: Value for Standard_NC24ads_A100_v4

File: assets/models/system/tiiuae-falcon-40b/spec.yaml (hunk 1 above)
Comment (BhavanaTumma) [reply assets/models/system/tiiuae-falcon-40b/spec.yaml:8]: Min SKUs added - offline conversation

**Review by BhavanaTumma — APPROVED** [review]
LGTM

**Comment by novaturient95** [comment]
valdn run with PR changes: https://github.com/Azure/azureml-assets/actions/runs/7486061627/job/20375661737?pr=2079

**Comment by novaturient95** [comment]
image

File: assets/models/system/gpt2-large/spec.yaml [hunk 3]
```diff
@@ -4,8 +4,8 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 97935fc1a406f447320c3db70fe9e9875dca2595
-  evaluation-min-sku-spec: 16|0|56|112
-  evaluation-recommended-sku: Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3, Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3, Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  evaluation-min-sku-spec: 4|0|28|56
```
Comment (novaturient95) [inline assets/models/system/gpt2-large/spec.yaml:7]: cpu-mem and disk space has reduced for this. Do we need to validate on in-min compute for this one?

File: assets/models/system/microsoft-deberta-xlarge/spec.yaml [hunk 4]
```diff
@@ -4,8 +4,8 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 971e67361eb2580900e26b7062470dee4773d324
-  evaluation-min-sku-spec: 16|0|56|112
-  evaluation-recommended-sku: Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3, Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3, Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  evaluation-min-sku-spec: 4|0|28|56
```
Comment (novaturient95) [inline assets/models/system/microsoft-deberta-xlarge/spec.yaml:7]: cpu-mem and disk space has reduced for this. Do we need to validate on in-min compute for this one?

File: assets/models/system/roberta-large/spec.yaml [hunk 5]
```diff
@@ -4,8 +4,8 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 716877d372b884cad6d419d828bac6c85b3b18d9
-  evaluation-min-sku-spec: 16|0|56|112
-  evaluation-recommended-sku: Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX4mds, Standard_F8s_v2, Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E4s_v3, Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3, Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3, Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4, Standard_ND40rs_v2
+  evaluation-min-sku-spec: 4|0|28|56
```
Comment (novaturient95) [inline assets/models/system/roberta-large/spec.yaml:7]: cpu-mem and disk space has reduced for this. Do we need to validate on min compute for this one?

Comment (sarthaks95) [reply assets/models/system/roberta-large/spec.yaml:7]: Validation exp - https://ml.azure.com/experiments/id/826c38b2-9d5a-4261-ba56-57b918a6729f?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-model-evaluation-rg/providers/Microsoft.MachineLearningServices/workspaces/aml-shared-model-evaluation-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

File: assets/models/system/microsoft-deberta-xlarge/spec.yaml (hunk 4 above)
Comment (sarthaks95) [reply assets/models/system/microsoft-deberta-xlarge/spec.yaml:7]: Validation exp - https://ml.azure.com/experiments/id/826c38b2-9d5a-4261-ba56-57b918a6729f?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-model-evaluation-rg/providers/Microsoft.MachineLearningServices/workspaces/aml-shared-model-evaluation-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

File: assets/models/system/gpt2-large/spec.yaml (hunk 3 above)
Comment (sarthaks95) [reply assets/models/system/gpt2-large/spec.yaml:7]: Validation exp - https://ml.azure.com/experiments/id/826c38b2-9d5a-4261-ba56-57b918a6729f?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-model-evaluation-rg/providers/Microsoft.MachineLearningServices/workspaces/aml-shared-model-evaluation-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

**Review by BhavanaTumma — APPROVED** [review]
Approved

**Pull request merged**